### ISSUE TITLE
Adding a static chart to a new worksheet in an existing xlsx

### ISSUE THREAD

**Issue by robesmithjr** [dfinke/ImportExcel#542]
Hi,

Great module, really useful.  I have been able to generate a few charts based on pivot tables, but I also need to be able to add some static clustered column charts based on simply Date and Hours.  I am able to use the  ColumnChart function to generate a ColumnChart from my custom object (random filename) this does not appear to support the passthru parameter, but have been unable to use the Chartdefinition to add the static chart after the pivotcharts have been created into my generated spreadsheet.

I tried running a few of the Example\Chart samples and they exhibit a similar behavior (chart is there, but no data), some header errors.

So basically, I want to export some data to separate worksheets and draw PivotCharts (working), then append some data into a new worksheet (also working) and generate a chart from it (have not been able to get this working).  

Thanks for your help.

Rob

**Comment by robesmithjr** [comment]
Hi,  

I made a few edits to charting.ps1 to facilitate specifying an existing excel file along with a worksheet name.  If no filename is supplied, the chart opens with a temp file name.
